artemis/reporting/modules/domain_expiration_scanner/reporter.py

```diff
@@ -0,0 +1,74 @@
+import os
+from typing import Any, Callable, Dict, List
+
+from artemis.reporting.base.language import Language
+from artemis.reporting.base.normal_form import (
+    NormalForm,
+    get_domain_normal_form,
+    get_domain_score,
+)
+from artemis.reporting.base.report import Report
+from artemis.reporting.base.report_type import ReportType
+from artemis.reporting.base.reporter import Reporter
+from artemis.reporting.base.templating import ReportEmailTemplateFragment
+from artemis.reporting.utils import get_top_level_target
+
+
+class DomainExpirationScannerReporter(Reporter):
+    CLOSE_DOMAIN_EXPIRATION_DATE = ReportType("close_domain_expiration_date")
+
+    @staticmethod
+    def create_reports(task_result: Dict[str, Any], language: Language) -> List[Report]:
+        if task_result["headers"]["receiver"] != "domain_expiration_scanner":
+            return []
+
+        if not task_result["status"] == "INTERESTING":
+            return []
+
+        if not isinstance(task_result["result"], dict):
+            return []
+
+        data = task_result["result"]
+        exp_date = data["expiration_date"]
```

Comment:
if the variables differ by type, that should be singified not by their shortened name, but by e.g. suffix